### PR TITLE
Added most of the ValueList Enums

### PR DESCRIPTION
**Pull request by Ultraporing**
I did leave out entries of the type "valuelist "foo"" without values.
Also updated some of the existing Enums.

### CODE REVIEW

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Act.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Act.cs:1]: Make sure to move using statements into the namespace to keep things consistent

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/BigQualifier.cs [hunk 2]
```diff
@@ -0,0 +1,313 @@
+﻿using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/BigQualifier.cs:1]: Move

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/DamageSourceType.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/DamageSourceType.cs:1]: Move

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Handedness.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Handedness.cs:1]: Move

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/PenaltyPreciseQualifier.cs [hunk 5]
```diff
@@ -0,0 +1,615 @@
+using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/PenaltyPreciseQualifier.cs:1]: Move

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/PreciseQualifier.cs [hunk 6]
```diff
@@ -0,0 +1,317 @@
+﻿using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/PreciseQualifier.cs:1]: Move

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Qualifier.cs [hunk 7]
```diff
@@ -0,0 +1,47 @@
+﻿using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Qualifier.cs:1]: Move

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Relation.cs [hunk 8]
```diff
@@ -0,0 +1,23 @@
+﻿using System.Runtime.Serialization;
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Relation.cs:1]: Move

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/YesNo.cs [hunk 9]
```diff
@@ -0,0 +1,11 @@
+﻿/// <summary>
+/// The YesNo
+/// </summary>
+namespace bg3_modders_multitool.Enums.ValueLists
+{
+    public enum YesNo
+    {
+        No,
```
Comment (ShinyHobo) [inline bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/YesNo.cs:8]: I believe I use a boolean converter to handle this, but this is fine

**Review by ShinyHobo — CHANGES_REQUESTED** [review]
Excellent work, just minor consistency points on the include statements. 

The reason why I don't just automatically generate the enums based on what is required at runtime is because I always intended on the GOE to be more. The plan was to extend it to allow you to create new GameObjects, but I never got around to it.

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Relation.cs (hunk 8 above)
Comment (Ultraporing) [reply bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/Relation.cs:1]: Will do tomorrow. Didn't pay attention when it added the using. Not gonna answer every one of the "Move" conversations. Once one is done I just resolve.

File: bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/YesNo.cs (hunk 9 above)
Comment (Ultraporing) [reply bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/YesNo.cs:8]: > I believe I use a boolean converter to handle this, but this is fine

It looked literally like this. Tbh no clue why they do such a thing.

**Comment by Ultraporing** [comment]
> Excellent work, just minor consistency points on the include statements.
> 
> The reason why I don't just automatically generate the enums based on what is required at runtime is because I always intended on the GOE to be more. The plan was to extend it to allow you to create new GameObjects, but I never got around to it.

Cant close this change request.
Also did add a global rule config that should take care of it now.

**Review by ShinyHobo — APPROVED** [review]

**Pull request merged**